### PR TITLE
Medborg Crew Monitor Upgrade

### PR DESCRIPTION
**Pull request by Surrealaser**
### About The Pull Request
Integrates an item action for remote crew monitor access with the crew locator upgrade for the medborg.

### Why It's Good For The Game
The important second part of the crew pinpointer; as a result, medborgs no longer have to glue themselves to fixed crew monitor consoles in order to do the paramedicine job they excel at, which is boring and lame.

### Changelog
🆑
add: Remote access to crew monitoring is now integrated with the crew locator upgrade for Mediborgs.
/🆑

### CODE REVIEW

**Comment by carlarctg** [comment]
can't you just merge this with the crew pinpointer upgrade? there's already like four other upgrade modules solely for the medborg, and it's getting kind of dumb and is a hassle to put into the borg, too

**Comment by Surrealaser** [comment]
@carlarctg I actually might do just that. I agree, the upgrade spam is kind of obnoxious. I actually wanted to include this as a default property of the medborg but figured that might be too controversial.

**Comment by Surrealaser** [comment]
@carlarctg Changed it per your request.

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -561,31 +561,49 @@
...
 	if (.)
+		icon_state = "pinpointer_crew"
+		crew_monitor.Remove(R)
+		QDEL_NULL(crew_monitor)
+		R.verbs -= /mob/living/silicon/robot/proc/cmd_robot_crewmonitor
 		var/obj/item/pinpointer/crew/PP = locate() in R.module
-		if (PP)
-			R.module.remove_module(PP, TRUE)
+		R.module.remove_module(PP, TRUE)
+		to_chat(world, "<span class='warning'>crew_monitor: [crew_monitor].</span>")
```
Comment (nemvar) [inline code/game/objects/items/robot/robot_upgrades.dm]: Delet dis

Comment (Surrealaser) [reply code/game/objects/items/robot/robot_upgrades.dm]: Ah fuck. Forgot about that debug. Thanks.

Comment (Surrealaser) [reply code/game/objects/items/robot/robot_upgrades.dm]: Fixed

**Comment by ExcessiveUseOfCobblestone** [comment]
button only no verb

**Comment by ExcessiveUseOfCobblestone** [comment]
really could just be a portable crew monitor module tho

**Comment by Surrealaser** [comment]
@ExcessiveUseOfCobblestone Too clumsy to merely add it as a PCM IMO. But sure, I can remove the verb.

**Comment by ExcessiveUseOfCobblestone** [comment]
what do you mean too clumsy...?

**Comment by Surrealaser** [comment]
@ExcessiveUseOfCobblestone Yet another object among many to juggle. Personally I would prefer it to be hands free for QoL purposes, and it makes sense that you have a direct remote connection to the crew monitor information as a borg. Further, it makes sense to integrate it with the pinpointer as it an integral pair that need each other. Having it available as a separate upgrade adds to the obnoxious upgrade spam mentioned by @carlarctg . 

The only way I wouldn't mind this is if Borgs started off with a hand held monitor by default.

**Comment by ExcessiveUseOfCobblestone** [comment]
I can't remember off the top of my head but if PCM lose the UI if you put it in say your pocket then medborgs should have this restriction as well imo since the only thing that balances this feature is the fact that you have to constantly stare at the thing.

If not then sure that's fine

**Comment by Surrealaser** [comment]
@ExcessiveUseOfCobblestone Not sure, but even if the UI went away when stashed in the pocket, having it as a non-default researchable upgrade is IMO adequate downside for not needing it in the hand. I mean it's not like there is a substantial benefit from being able to see it in real time, versus taking it out as needed; the difference is mainly QoL.

**Comment by Names-Are-Hard** [comment]
Portable crew monitors do work when pocketed. @ExcessiveUseOfCobblestone

**Comment by Surrealaser** [comment]
@nemvar Is this ready to go?

**Comment by ExcessiveUseOfCobblestone** [comment]
> @ExcessiveUseOfCobblestone Not sure, but even if the UI went away when stashed in the pocket, having it as a non-default researchable upgrade is IMO adequate downside to balance out not needing it in the hand. I mean it's not like there is a substantial benefit from being able to see it in real time, versus taking it out as needed; the difference is mainly QoL.

QoL is balance for this dumb feature (crew monitors in general, not what ur coding now), since it's literally only balanced by the ease of checking the thing and continually checking it, which something a borg player is nuts enough to do :omegalol:

**Pull request merged**